src/app/machines/views/MachineDetails/MachineNetwork/NetworkFields/NetworkFields.tsx

```diff
@@ -1,31 +1,39 @@
+import { useEffect } from "react";
+
 import { useFormikContext } from "formik";
+import { isIPv4 } from "is-ip";
 import { useSelector } from "react-redux";
 import * as Yup from "yup";
```

Comment:
It looks like the code imports `Yup` here for validation, but there is no validation schema provided below. (I might be missing something.)

Reply:
Ah - you'll need to expand line 40 in that file to see it, it is there though!